cosmos/providers/dbt/core/operators.py

```diff
@@ -234,45 +223,33 @@ def run_command(self, cmd, env):
         self.exception_handling(result)
         return result
 
-    def build_and_run_cmd(self, env: dict, cmd_flags: list = None):
+    def build_and_run_cmd(
+        self, context: Context, cmd_flags: list[str] | None = None
+    ) -> SubprocessResult:
         create_default_profiles(DBT_PROFILE_PATH)
+        create_default_profiles()
```

Comment:
@MikeWallis42 can't tell if this was an accidental addition or not - can it be removed?

Reply:
@jlaneve apologies it was a rogue duplication, that function expects the path to be passed in now so it would have generated an error, I've removed it.